VideoLocker/src/main/java/org/edx/mobile/module/prefs/UserPrefs.java

```diff
@@ -51,20 +52,21 @@ public boolean isDownloadOverWifiOnly() {
      */
     public File getDownloadFolder() {
         ProfileModel profile = getProfile();
-        
-        File android = new File(Environment.getExternalStorageDirectory(), "Android");
-        File downloadsDir = new File(android, "data");
-        File packDir = new File(downloadsDir, context.getPackageName());
-        File edxDir = new File(packDir, profile.username);
-        edxDir.mkdirs();
+        File videosDir = new File(context.getExternalFilesDir(null).getParent(), AppConstants.Folders.VIDEOS);
```

Comment:
As for what to do if the external storage is not available when updating the database, I suppose there are three options for handling this as well:

1. Don't migrate the videos at all.
2. Listen for the `ACTION_MEDIA_MOUNTED` broadcast, and do the migration at that point. This would probably be the 'proper' course of action (to make everything available as soon as technically possible), but it would also be the most complex logic.
3. Fail the database upgrade until such a time as when the external storage may be mounted. This may leave the app in an unusable state if the external storage is not mounted for an extended period of time, so it's probably not a good idea.